.eslintrc.yaml

```diff
@@ -35,7 +35,7 @@ rules:
     [error, { alphabetize: { order: "asc" }, groups: [["builtin", "external", "internal"], "parent", "sibling"] }]
   no-async-promise-executor: off
   # This isn't a real module, just types, which apparently doesn't resolve.
-  import/no-unresolved: [error, { ignore: ["express-serve-static-core"] }]
+  import/no-unresolved: [error, { ignore: ["express-serve-static-core", "code-server"] }]
```

Comment:
[asking to learn/understand]: why is this needed?

Reply:
eslint can't find the types because `code-server` is an alias so I just ignored it because `tsc` resolves it just fine. But the real solution is to add the alias to eslint, so I'll do that.